156613344 redesign tasks

DevStarlight opened the pull request:
- Video card verified.
- Update video upload page drop-box aspect ratio and text colors.
- Updated video upload page styles.
- Update video-card verified icon to use material icon; update color and size.

CodingCarlos committed:
Video Card verified

javierLiarte committed:
Update video upload page drop-box aspect ratio and text colors

- Move up .panoramic CSS style from video-detail-edit.view.scss to sass/style.scss
- Updated video upload page styles

javierLiarte committed:
Merge branch '156613344_redesign_tasks' of github.com:Videona/platform-frontend into 156613344_redesign_tasks

javierLiarte committed:
Update video-card verified icon to use material icon; update color and size

javierLiarte reviewed (approved):
Seems all ok here. Also QA is passed

javierLiarte commented:
pending to update versioning and merge to dev

javierLiarte committed:
Setup for v0.7.7
- Fixed verified icon lost in gallery video cards
- Fixed UI issues on video upload page drop-box area
